package.json

```diff
@@ -113,6 +111,9 @@
     "uuid": "^3.3.3",
     "winston": "^3.2.1"
   },
+  "dependencies": {
+    "geospatialdraw": "0.5.7"
```

Comment:
❗️ the current version on npm is `0.5.6`

Reply:
:+1: I need to wait until my changes in geospatialdraw have been pushed to npm, including a version bump

Reply:
Marking as resolved because 0.5.7 is released